docs/docs/reference/other-new-features/safe-initialization.md

```diff
@@ -141,11 +144,19 @@ field points to an initialized object may not later point to an
 object under initialization. As an example, the following code will be rejected:
 
 ``` scala
```

Comment:
Should we add `sc-fail` then?

Reply:
Apparently, this code breaks only at runtime.

Reply:
Is this expected? cc @liufengyun